test/lib/ex_aws/s3/download_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule ExAws.S3.DownloadTest do
```

Comment:
Thanks!

Haven't really used bypass before, it could likely be a valuable addition to this library's test suite given that it's an API client and all.